tutorials/run-meteor-on-google-app-engine.md

```diff
@@ -142,20 +142,33 @@ running the following command:
 
 1. Replace the contents of the `Dockerfile` file with the following:
 
-        FROM gcr.io/google_appengine/nodejs
+        FROM launcher.gcr.io/google/nodejs
+        RUN install_node v4.6.2
```

Comment:
Is there any way to avoid hard-coding a single version of Node.js?

Reply:
Could we use Node8? 6 Will go end of life in a few months.